pkgs/development/python-modules/dbx/default.nix

```diff
@@ -0,0 +1,58 @@
+{ buildPythonPackage
+, fetchPypi
+, databricks-cli
+, scipy
+, pathpy
+, tqdm
+, mlflow
+, azure-identity
+, ruamel_yaml
+, emoji
+, cookiecutter
+, retry
+, azure-mgmt-datafactory
+, azure-mgmt-subscription
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "dbx";
+  version = "0.4.1";
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    dist = "py3";
+    python = "py3";
+    sha256 = "PzLxks8DPVXKF63S10+/9+gqxnXd1MTlDIimKPJq12M=";
+  };
+
+  propagatedBuildInputs = [
+    databricks-cli
+    scipy
+    pathpy
+    tqdm
+    mlflow
+    azure-identity
+    ruamel_yaml
+    emoji
+    cookiecutter
+    retry
+    azure-mgmt-datafactory
+    azure-mgmt-subscription
+  ];
+
+  doCheck = false;
```

Comment:
Missing `pythonImportsCheck` and the repo contains tests. Why can't we run them? If they are not in the tarball we should fetch from GitHub.